dev/com.ibm.ws.injection_fat/publish/servers/com.ibm.ws.injection.fat.EnvEntryServer/server.xml

```diff
@@ -1,6 +1,6 @@
 <server>
     <featureManager>
-        <feature>componentTest-1.0</feature>
+        <feature>componenttest-1.0</feature>
```

Comment:
For future reference: last week I reviewed something for Andy that should fix the case mattering in the feature name for the repeatTest stuff.

Reply:
:-) Yes, it was Andy that helped me figure out why it wasn't working.  He said he would get it fixed, but I thought it best going forward to just fix it.